src/gui/gui_msgs.h

```diff
@@ -37,45 +37,62 @@ constexpr char help_msg[] =
 
-  --working-dir <path> Set working directory to <path>. DOSBox will act as if
-                       started from this directory.
+  --opencaptures           Launches some external program (I don't even know...)
 
-  -fullscreen          Start in fullscreen mode.
+  --set <property>=<value> Sets a configuration property.
+                           Example: --set mididevice=fluidsynth
 
-  -lang <langfile>     Start with the language specified in <langfile>.
+  --socket <num>           Runs nullmodem on the specified socket number
 
-  --list-glshaders     List available GLSL shaders and their directories.
-                       Results are useable in the "glshader = " config setting.
+  -c <command>             Run the specified DOS command before running FILE.
+                           Multiple commands can be specified.
 
-  -machine <type>      Emulate a specific type of machine. The machine type has
-                       influence on both the emulated video and sound cards.
-                       Valid choices are: hercules, cga, cga_mono, tandy,
-                       pcjr, ega, svga_s3 (default), svga_et3000, svga_et4000,
-                       svga_paradise, vesa_nolfb, vesa_oldvbe.
+  --noautoexec             Don't perform any [autoexec] actions.
 
-  -exit                Exit after the DOS program specified by FILE has ended.
+  --noprimaryconf          Don't read settings from the primary configuration file
+                           located in your user folder.
 
-  -h, --help           Print this help message and exit.
+  --nolocalconf            Don't read settings from "dosbox.conf" if present in
+                           the current working directory.
 
-  -v, --version        Print version information and exit.
+  --conf <configfile>      Start with the options specified in <configfile>.
+                           Multiple configfiles can be specified.
+
+  --working-dir <path>     Set working directory to <path>. DOSBox will act as if
+                           started from this directory.
+
+  --fullscreen             Start in fullscreen mode.
+
+  --lang <langfile>        Start with the language specified in <langfile>.
+
+  --list-glshaders         List available GLSL shaders and their directories.
+                           Results are useable in the "glshader = " config setting.
+
+  --machine <type>         Emulate a specific type of machine. The machine type has
+                           influence on both the emulated video and sound cards.
+                           Valid choices are: hercules, cga, cga_mono, tandy,
+                           pcjr, ega, svga_s3 (default), svga_et3000, svga_et4000,
+                           svga_paradise, vesa_nolfb, vesa_oldvbe.
+
+  --exit                   Exit after the DOS program specified by FILE has ended.
+
+  -h, --help               Print this help message and exit.
+
+  -v, --version            Print version information and exit.
```

Comment:
Can we remove the two lines below? The man page is either going away or it will contain the same info, same for the README, plus that path is distro/installations specific. Better to just rid of them.

Reply:
Yeah, I was eyeing those lines suspiciously.  Removed.